### ISSUE TITLE
Kemasan Kerajinan 

### ISSUE THREAD

**Issue by dvanysukma** [dvanysukma/Kemasan-Kerajinan#1]
KEMASAN PRODUK KERAJINAN BAHAN KERAS

![kemasan](https://user-images.githubusercontent.com/114471282/205826191-a6f5d580-e935-4d3a-8be6-b864bfe7b2a3.jpg)

Kemasan secara umum dimaksudkan adalah sebagai bagian terluar yang membungkus suatu produk dengan tujuan untuk melindungi produk dari cuaca, guncangan dan benturan-benturan terhadap benda lain. Setiap bentuk barang benda yang membungkus suatu benda di dalamnya dapat disebut dengan kemasan sejauh hal tersebut memang bermanfaat untuk melindungi isinya. Untuk menampilkan kesan dan pandangan terhadap suatu isi produk, maka packaging biasanya dibentuk atau di desain sedemikian rupa, sehingga pesan yang akan disampaikan akan dapat ditangkap oleh pemakai produk dengan baik.

Penyajian sebuah produk kerajinan sangat dilakukan merupakan bagian penting dari sebuah karya. Di mana penyajian produk kerajinan yang dilakukan dengan bentuk kemasan. Kemasan yang dilakukan pada sebuah produk sangat menentukan apakah produk tersebut layak dikatakan memiliki kualitas lebih atau tidak. Kemasan produk adalah bagian pembungkus dari suatu produk yang ada di dalamnya. Pengemasan tersebut merupakan salah satu cara untuk mengawetkan atau memperpanjang umur dari produk-produk pangan atau makanan yang terdapat di dalamnya.

**Fungsi Kemasan**
Meski awalnya kemasan hanya berfungsi sebagai wadah dan menjadi keamanan produk, tapi seiring perkembangan zaman fungsi kemasan ikut berubah. Dikutip dari situs Kementerian Pendidikan dan Kebudayaan (Kemdikbud), dilihat dari fungsinya kemasan memiliki bebeberapa fungsi utama. Berikut fungsi utama kemasan:

- Menjual produk
- Melindungi produk
- Memudahkan penggunakan produk
- Memperindah penampilan produk

Keempat fungsi tersebut sangat penting dan perlu diperhatikan agar bisa meningkatkan daya jual produk. Kemasan sebagai pelengkap produk dengan tujuan agar produk terlihat lebih menarik. Untuk bahan kemasan produk beragam, seperti logam, plastik, kayu, serat alam, kardus, kaca, mau mika.

**Tujuan Kemasan**
Suatu kemasan produk dinilai ideal jika memenuhi dua syarat, yaitu syarat proteksi produk dan syarat estetika. Kemasan produk kerajinan limbah keras harus dapat melindungi produk dari risiko kerusakan, kotoran, benturan, dan kondisi buruk sejenisnya. Selain itu, kemasan produk juga harus memiliki nilai estetika untuk menarik perhatian konsumen agar membeli produk tersebut. Karena itu, kemasan perlu dibuat dengan desain memikat. 

Ada sejumlah tujuan pembuatan kemasan untuk produk kerajinan dari bahan limbah keras. Sebagaimana dikutip dari Pembuatan Kerajinan Bahan Limbah Keras (2020) yang ditulis Henni Ratnasusanti, berikut ini sejumlah tujuan dan fungsi dari kemasan produk kerajinan limbah keras:

- Kemasan untuk mewadahi produk selama distribusi dari penjual hingga ke konsumen.
- Kemasan untuk melindungi hasil kerajinan limbah keras dari sinar ultraviolet yang berisiko merusak produk, melindungi dari panas, kelembaban udara, benturan, dan lain sebagainya.
- Kemasan untuk menunjukkan identitas produk. Ia menampilkan informasi kepada konsumen melalui label yang terdapat pada kemasan. Identitas ini juga bisa menjadi pembeda dengan produk dari kompetitor.
- Kemasan untuk memberikan nilai tambah estetika dari produk sehingga dapat meningkatkan daya tarik calon pembeli.
- Kemasan dapat menjadi sarana informasi dan iklan. Di bungkus kemasan, produsen dapat menambahkan alamat situs web, sosial media, layanan konsumen, dan lain sebagainya.

Selain tujuan di atas, beberapa tujuan lainnya yaitu:
- Physical Production : Sebuah kemasan bertujuan untuk melindungi objek dari suhu, getaran, guncangan, tekanan dan sebagainya.
- Barrier Protection : Kemasan mempunyai tujuan untuk melindungi dari halangan oksigen uap air, debu, dan lain sebagainya.
- Containment or Angglomeration : Untuk efisiensi transportasi dan penanganan, maka produk dengan ukurang kecil biasanya dikelompokkan dalam satu paket kemasan.
- Information Transmission : Informasi tenang menggunakan transportasi, daur ulang, atau membuang paket produk sering terdapat pada kemasan atau label
- Reducing Theft : Jika ada kemasan yang tidak bisa ditutup kembali atau akan rusak secara fisik sangan membantu dalam pencegahan pencurian. Paket juga termasuk memberikan kesempatan sebagai perangkat anti-pencurian.
- Convenience : Fitur yang menjadi penambah kenyamanan dalam distribusi, penanganan, penjualan, tampilan, pembukaan, kembali penutup, penggunaan dan digunakan kembali.
- Marketing : Kemasan dan label bisa dipakai oleh pemasar untuk mendorong calon pembeli untuk membeli produk.

**Tujuan Kemasan**
_Jenis Kemasan Berdasarkan Struktur Isi_
- Kemasan Primer yakni bahan kemas langsung mewadahi bahan pangan (kaleng susu, botol minuman, dan lain sebagainya)
- Kemasan Sekunder ialah kemasan yang berfungsi melindungi kelompok kemasan lainnya, seperti misalnya kotak karton untuk kaleng susu, kotak kayu untuk wadah buahyang dibungkus dan lain sebagainya.
- Kemasan Tersier dan Kuarter yaitu kemasan yang dibutuhkan untuk menyimpan barang atau produk selama pengiriman.

_Jenis Kemasan Berdasarkan Frekuensi Pemakaian_
- Kemasan Sekali Pakai (Disposable) ialah jenis kemasan ini adalah kemasan yang langsung dibuang setelah sekali pakai. Contohnya bungkus plastik, bungkus daun dan lain sebagainya.
- Kemasan yang Dapat Dipakai Berulang Kali (Multi Trip) yakni kemasan jenis ini umumnya tidak dibuang oleh konsumen, namun dikembalikan lagi pada agen penjual untuk kemudian dimanfaatkan ulang. Contohnya botol minuman dan lain sebagainya.
- Kemasan yang Tidak Dibuang (Semi Disposable) yaitu Jenis kemasan ini biasanya digunakan untuk kepentingan lain di rumah konsumen setelah digunakan. Contohnya kaleng biskuit dan lain sebagainya.

_Jenis Kemasan Berdasarkan Tingkat Kesiapan Pakai_
- Kemasan Siap Pakai yaitu jenis ini yaitu jenis bahan kemas yang siap diisi dengan bentuk yang telah sempurna sejak keluar dari pabrik. Contohnya adalah botol, kaleng dan lain sebagainya.
- Kemasan Siap Dirakit yakni jenis kemasan yang masih memerlukan tahap perakitan sebelum pengisian, misalnya kaleng dalam bentuk lempengan dan silinder fleksibel, wadah yang terbuat dari kertas, foil atau plastik.

**Kemasan Kertas,** merupakan kemasan fleksibel yang pertama sebelum ditemukannya plastik dan aluminium foil. Saat ini kemasan kertas masih banyak digunakan dan mampu bersaing dengan kemasan lain seperti plastik dan logam karena harganya yang murah, mudah diperoleh, dan penggunaannya yang luas. Kelemahan kemasan kertas untuk mengemas adalah sifanya yang sensitif terhadap air dan mudah dipengaruhi oleh kelembaban udara lingkungan.

**Kemasan Kayu,** merupakan bahan pengemas tertua yang diketahui oleh manusia dan secara tradisional digunakan untuk mengemas berbagai macam produk padat seperti barang antik dan emas, keramik, dan kain. Kayu adalah bahan baku dalam pembuatan palet, peti atau kotak kayu di negara-negara yang mempunyai sumber kayu alam dalam jumlah banyak. 
Kemasan Plastik, merupakan kemasan yang paling banyak kita temui adalah kemasan plastik. Beberapa jenis kemasan plastik yang dikenal adalah polietilen, polipropilen, poliester, nilon, dan vinil film.
